src/docs/Components/Form/Checkbox/index.mdx

```diff
@@ -4,13 +4,15 @@ order: 1
 description: 'Checkboxes are used to select one or multiple options in a list. They usually find their place in forms and are also used to accept some mentions'
 links:
   vue:
-    link: 'https://adeo.github.io/mozaic-vue/?path=/story/components-mcheckbox--try-it'
+    link: 'https://adeo.github.io/mozaic-vue/?path=/story/components-mcheckbox'
```

Comment:
The link is broken.
The good url is => https://adeo.github.io/mozaic-vue/?path=/story/components-form-mcheckbox--default